Styles/Card.css

```
@@ -0,0 +1,65 @@
+/*.content__box__wgbox,
+  .content__box__conduentbox,
+  .content__box__kmabox */
+.card {
+  display: flex;
+  background-color: rgb(255, 255, 255);
+  width: 100%;
+}
+
+/*.content__box__wgbox ul,
+  .content__box__conduentbox ul,
+  .content__box__kmabox ul */
+.card ul {
```

To be more specific, I would add a class here instead: 
```suggestion
.card__list {
```